.github/workflows/release.yml

```diff
@@ -9,38 +9,43 @@ on:
         description: 'Version number (e.g., v1.0.0, v1.0.0-pre, v1.0.0-pre1)'
         required: true
       skip_tests:
-        description: 'Skip QA acceptance tests, define value to `true` to explicitly skip'
+        description: 'Set value to `true` to skip QA acceptance tests, default is `false`'
+        default: 'false'
       use_existing_tag:
-        description: 'Set value to `true` to use an existing tag for this release version, default is `false`'
+        description: 'Set value to `true` to use an existing tag for the release process, default is `false`'
+        default: 'false'
```

Comment:
were you able to test re-running a failing release, e.g. because tests failed and wanted to run again?

what happens if some PRs needs to be done, do we need to manually delete the tag and start again?   If these PRs have changelog entries, will the changelog file be updated?

Reply:
> were you able to test re-running a failing release, e.g. because tests failed and wanted to run again?

Updated the the PR description with the scenarios I tested. As of the example you mention it would be the case of running the release with `use existing tag`, as you want to rerun the tests and release + jira steps if successful.

> what happens if some PRs needs to be done, do we need to manually delete the tag and start again? If these PRs have changelog entries, will the changelog file be updated?

Yes for this case I do not see another alternative. The tag would be deleted, from there we can trigger the changelog generation workflow manually (which will update the unreleased section and remove the header of the deleted tag), and then you can trigger a new release action.

As a side note, thinking these questions might be nice to add in a FAQ section in the releasing document, WDYT?

Reply:
yeah, we can add it to FAQ.

about last case, my understanding is that tag and commits to master are done, so the changelog version for that tag is changed.    So even if the tag is deleted when the process to update Changelog runs, it will detect that the version is already in the Changelog and it won't update the entries for the PRs neede to fix the issues merged after .

Reply:
> So even if the tag is deleted when the process to update Changelog runs, it will detect that the version is already in the Changelog and it won't update the entries for the PRs needed to fix the issues merged after

This statement is actually not correct. Will explain with an example:

- we have the tag created for v1.16.0 and and adjusted the changelog header, but tests are failing
- Some additional commits are then merged into master with new changelog entries
- Now we want to delete v1.16.0 tag because we want to introduce new changes in the release
- Once we delete the tag, when running the update-changelog workflow, as seen in [update-changelog-unreleased-section.sh](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/scripts/update-changelog-unreleased-section.sh#L24) it preserves the content of the changelog using the last tag defined in the repo (in this case v1.15.3), meaning the unreleased section will have all entries from v1.15.3 to the current commit.